projects/aws-observability/aws-otel-collector/README.md

```diff
@@ -0,0 +1,11 @@
+## **AWS Distro for OpenTelemetry Collector**
+
+AWS Distro for OpenTelemetry Collector ([ADOT Collector](https://github.com/aws-observability/aws-otel-collector)) is an AWS supported version of the upstream OpenTelemetry Collector and is distributed by Amazon. It supports the selected components from the OpenTelemetry community.
+
+### Updating
+
+1. Review [releases](https://github.com/aws-observability/aws-otel-collector/releases)
+1. Update the `GIT_TAG` file to have the new desired version based on the upstream release tags.
```

Comment:
Should we mention the update digests?

Reply:
Yes, good callout. I just re-organized the `README` and added more contents. I still plan to do one more update to this file once I have all other pieces sorted out (CDK changes, update frequency, etc.)